include/aspect/gravity_model/radial.h

```diff
@@ -135,6 +135,52 @@ namespace aspect
         double magnitude_at_bottom;
 
     };
+
+
+
+    template <int dim>
```

Comment:
Please add doxygen documentation above the class name about what this class does. You can use the text you wrote at the bottom of the source file, but bring it into the doxygen format (/** ... */). This will be used in the online documentation (here the example for `RadialLinear`: https://aspect.geodynamics.org/doc/doxygen/classaspect_1_1GravityModel_1_1RadialLinear.html.